src/scss/override/_alerts.scss

```diff
@@ -1,37 +1,73 @@
-﻿// Override
+//
+// Static Resources
+//
+$alert-icon-base: 48px;
```

Comment:
48? I thought we had a mixin now for responsive spacings, or will this be 48px on all viewports?